### PR TITLE
dataMoverTopic not required anymore and defaults to empty string

### PR DESCRIPTION
**Pull request by KillianG**
Do not create a consumer if no topic provided

Issue: BB-565

### CODE REVIEW

File: extensions/replication/ReplicationConfigValidator.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ const joiSchema = joi.object({
         bootstrapList: bootstrapListJoi,
     },
     topic: joi.string().required(),
-    dataMoverTopic: joi.string().required(),
+    dataMoverTopic: joi.string().default(''),
```
Comment (welansari) [inline extensions/replication/ReplicationConfigValidator.js]: maybe we should just leave it undefined ?

**Review by welansari — APPROVED** [review]

**Review by benzekrimaha — APPROVED** [review]

File: extensions/replication/ReplicationConfigValidator.js (hunk 1 above)
Comment (francoisferrand) [inline extensions/replication/ReplicationConfigValidator.js]: can we make the topic required if lifecycle transition is enabled?

**Pull request merged**